process/factory/shard/preProcessorsContainerFactory_test.go

```diff
@@ -366,7 +384,9 @@ func TestPreProcessorsContainerFactory_CreateSCRPreprocessor(t *testing.T) {
 		t.Parallel()
 
 		args := createMockPreProcessorsContainerFactoryArguments()
-		args.SmartContractResultPreProcessorCreator, _ = preprocess.NewSmartContractResultPreProcessorFactory()
+		runTypeComps := processMocks.NewRunTypeComponentsStub()
```

Comment:
You don't need to do this. `createMockPreProcessorsContainerFactoryArguments` already does this and `SCResultsPreProcessorFactory` is already initialized with `NewSmartContractResultPreProcessorFactory`

Reply:
Right, deleted it